### PR TITLE
Trigger Assignment

### PR DESCRIPTION
**Pull request by Manish-Fexle-May2022**
Change Contact Status According To Account Status.........!!!

### CODE REVIEW

File: src/classes/AccountHelper.cls [hunk 1]
```diff
@@ -0,0 +1,15 @@
+public class AccountHelper {
+    public static void changeContactActiveStatusAccordingToRelatedAccount(List<Account> accList) {
+        List<Account> newAccList = new List<Account>();
+        for (Account acc : accList) {
+            if(acc.Status__C == 'Active') {
```
Comment (DilipJainDj) [inline src/classes/AccountHelper.cls:5]: Please add the condition to check that trigger is not running on the false update like will only run once status is moved to active from the other status

Comment (Manish-Fexle-May2022) [reply src/classes/AccountHelper.cls:5]: Ok Sir

File: src/classes/AccountHelper.cls [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+                newAccList.add(acc);
+            }
+        }
+        List<Contact> conList = [Select Id, Name, Status__c, AccountId From Contact Where AccountId in : newAccList AND Status__c != 'Active'];
+        for (Contact con : conList) {
+             con.Status__c = 'Active';
+        }
+        update conList;
```
Comment (DilipJainDj) [inline src/classes/AccountHelper.cls:13]: Add the check to verify that data is not null in the contact List.

Comment (DilipJainDj) [reply src/classes/AccountHelper.cls:13]: Try to use list.isEmpty() check

**Review by DilipJainDj — COMMENTED** [review]
Review the comments and perform the changes.

File: src/classes/AccountHelper.cls (hunk 2 above)
Comment (Manish-Fexle-May2022) [reply src/classes/AccountHelper.cls:13]: Ok Sir

**Pull request merged**